tsfc/spectral.py

```diff
@@ -44,14 +32,86 @@ def Integrals(expressions, quadrature_multiindex, argument_multiindices, paramet
 
     :returns: list of integral representations
     """
+    # Rewrite: a / b => a * (1 / b)
+    expressions = replace_division(expressions)
```

Comment:
This continues to feel like cargo-culting to me.  Maybe @FabioLuporini can comment, but is it really the case that compilers do a better job when given:

```c

A = B * (1/C);
```
?